js/script.js

```diff
@@ -0,0 +1,116 @@
+import {
+  $taskInput,
+  $addButton,
+  $taskList,
+  $searchInput,
+  $searchButton,
+} from "./elements.js";
+import {
+  showToastMessage,
+  createTaskElement,
+  sanitizeInput,
+  formatDate,
+} from "./utilities.js";
+
+let tasks = [];
+
+const addButtonHandler = () => {
+  const taskTitle = sanitizeInput($taskInput.value);
+  if (taskTitle) {
+    createTask(taskTitle);
+    showToastMessage("Task added successfully!");
+    $addButton.disabled = true;
+  } else {
+    showToastMessage("Please provide a valid title!");
+  }
+};
```

Comment:
Please make changes that are only related to the specific PR

Reply:
Bhai, I didn't do rebase properly. Now I think you won't see those inconvenience.
Thank you.